webapp-admin/src/components/SigningLog.js

```diff
@@ -30,12 +30,20 @@ class SigningLog extends Component {
 
     super(props)
     this.state = {
+        logs: [],
+        filterModels: [],
+        filterString: '',
         message: null,
         expanded: {models: true},
         query: '',
         startRow: 0,
         endRow: PAGINATION_SIZE,
+        totalLogs: 0,
+        authorityID: null,
     };
+    this.state.authorityID = this.props.selectedAccount.AuthorityID
```

Comment:
you could change line 42 to:
`authorityID: this.props.selectedAccount.AuthorityID`
and remove this line

Reply:
fixed